src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskInitiateByIdControllerTest.java

```diff
@@ -96,42 +98,485 @@ public void should_return_a_201_when_initiating_a_task_by_id() {
             .body(
                 "execution_type_code.description",
                 equalTo("The task requires a case management event to be executed by the user. "
-                        + "(Typically this will be in CCD.)")
+                            + "(Typically this will be in CCD.)")
+            )
+            .body("work_type_resource.id", equalTo("hearing_work"))
+            .body("work_type_resource.label", equalTo("Hearing work"))
+            .body("task_role_resources.size()", equalTo(3));
+
+        assertPermissions(
+            getTaskResource(result, "hearing-judge"),
```

Comment:
Maybe a clarification with Helen as per AC. Is this case-role? AC says it is and organisation-role should = judge